enos/provider/vbox.py

```diff
@@ -85,9 +89,10 @@ def init(self, config, calldir, force_deploy=False):
                             quiet_stderr=False)
         if force_deploy:
             v.destroy()
-        v.up()
+        # NOTE(matrohon) : force no parallel provisionning since it fails with
+        # libvrit/hostmanager plugins for vagrant
+        v._call_vagrant_command(['up', '--no-parallel'])
```

Comment:
Maybe it's better to force the `VAGRANT_NO_PARALLEL` in the constructor [[1](https://github.com/todddeluca/python-vagrant/blob/master/vagrant/__init__.py#L232)] rather than calling this private method.